modules/ROOT/pages/streaming.adoc

```diff
@@ -17,15 +17,14 @@ To understand how data streams are consumed, review the following points:
 
 * Data streams cannot be consumed more than once
 +
-In the following example, the flow writes the first file correctly, while the second file is created with empty content because each component that consumes a stream expects to receive a new stream. After the first File Write operation consumes the stream, the second File Write operation receives an empty stream. Thereby, the second operation has no content to write to a file.
+In the following example, the flow shows the HTTP Listener source that receives a POST method with a body payload to write to the files. The flow writes the first file correctly, while the second file is created with empty content because each component that consumes a stream expects to receive a new stream. After the first File Write operation consumes the stream, the second File Write operation receives an empty stream. Thereby, the second operation has no content to write to a file.
 +
 .Mule 3 Streaming: Writing a File
 image::mruntime-streaming-about-1.png[]
 +
 In the following example, something similar happens when you try to log the payload before and
 after a DataWeave transformation. The flow logs the payload before the Transform Message component, but does not
-log the resulting payload after the Transform Message component because the Logger component consumes
-and loads the stream into memory.
+log the resulting payload after the Transform Message component because the Logger component consumes the stream.
 Because the stream content is available in memory, the Transform Message component consumes the content and the second Logger component receives an empty stream.
```

Comment:
`Because the stream content is available in memory ` -> this is not true, it was consumed.

Reply:
Updated the sentence to its original content in the Mule 4 version.